metricbeat/docs/modules/elasticsearch/index.asciidoc

```diff
@@ -0,0 +1,23 @@
+////
+This file is generated! See scripts/docs_collector.py
+////
+
+[[metricbeat-metricset-elasticsearch-index]]
+=== Elasticsearch index metricset
+
+experimental[]
+
+include::../../../module/elasticsearch/index/_meta/docs.asciidoc[]
+
+
+==== Fields
+
+For a description of each field in the metricset, see the
+<<exported-fields-elasticsearch,exported fields>> section.
```

Comment:
This ID doesn't to exist `exported-fields-elasticsearch`?

Reply:
weird, as it is here: https://github.com/elastic/beats/pull/6881/files#diff-d43087d796b8b1d1ff48ec3561aeda60R3286